apps/css-workshop/src/pages/menu.astro

```diff
@@ -490,41 +507,44 @@ import Icon_ from '../components/Icon.astro';
-      <li class='iui-list-item iui-large' data-iui-actionable='true' role='menuitem' tabindex='0'>
-        <svg-placeholder class='iui-list-item-icon iui-header-menu-icon' aria-hidden='true'
-        ></svg-placeholder>
-        <span class='iui-list-item-content'>
-          <div>Model Alpha</div>
-          <div class='iui-list-item-description'>0x0987654321</div>
-        </span>
-      </li>
-      <li
-        class='iui-list-item iui-large iui-active'
-        data-iui-actionable='true'
+      <ListItem_
+        contentProps={{ class: 'iui-list-item-content' }}
+        size='large'
         role='menuitem'
-        aria-checked='true'
+        actionable
+        startIconProps={{ class: 'iui-header-menu-icon' }}
+      >
+        <svg-placeholder slot='start-icon'></svg-placeholder>
+        <div>Model Alpha</div>
+        <div class='iui-list-item-description'>0x0987654321</div>
+      </ListItem_>
+      <ListItem_
+        contentProps={{ class: 'iui-list-item-content' }}
+        size='large'
+        isActive
         tabindex='0'
+        role='menuitem'
+        actionable
+        startIconProps={{ class: 'iui-header-menu-icon' }}
       >
-        <svg-model class='iui-list-item-icon iui-header-menu-icon' aria-hidden='true'></svg-model>
-        <span class='iui-list-item-content'>
-          <div>Model Beta</div>
-          <div class='iui-list-item-description'>0x0123456789</div>
-        </span>
-        <svg-checkmark class='iui-list-item-icon' aria-hidden='true'></svg-checkmark>
-      </li>
-      <li class='iui-list-item iui-large' data-iui-actionable='true' role='menuitem' tabindex='0'>
-        <div class='iui-list-item-icon iui-header-menu-icon demo-picture'></div>
-        <span class='iui-list-item-content'>
-          <div>Model Lorem ipsum dolor sit amet, consectetur adipiscing elit</div>
-          <div class='iui-list-item-description'>0x0192837465</div>
-        </span>
-      </li>
+        <svg-model slot='start-icon'></svg-model>
+        <div>Model Beta</div>
+        <div class='iui-list-item-description'>0x0123456789</div>
+        <svg-checkmark slot='end-icon'></svg-checkmark>
+      </ListItem_>
+      <ListItem_
+        contentProps={{ class: 'iui-list-item-content' }}
+        size='large'
+        role='menuitem'
+        actionable
+        startIconProps={{ style: 'width: 24px; height: 24px' }}
+      >
+        <div class='iui-header-menu-icon demo-picture' slot='start-icon'></div>
+        <div>Model Lorem ipsum dolor sit amet, consectetur adipiscing elit</div>
+        <div class='iui-list-item-description'>0x0192837465</div>
+      </ListItem_>
       <li class='iui-menu-divider' role='separator'></li>
-      <li class='iui-list-item' data-iui-actionable='true' role='menuitem' tabindex='0'>
-        <span class='iui-list-item-content'>
-          <div>My projects</div>
-        </span>
-      </li>
+      <ListItem_ actionable tabindex='0' role='menuitem'>My projects</ListItem_>
```

Comment:
I have added the `contentProps` here. I think in the skeletons `iui-content` is used instead of `iui-list-item-content`.

Reply:
I noticed these `tabindex` in the `MenuItem` React component are set to 0 based on whether the menu item is selected or not. Not sure if these here in `menu.astro` are the same thing.

Reply:
Menu is supposed to use [roving tabindex](https://developer.mozilla.org/en-US/docs/Web/Accessibility/Keyboard-navigable_JavaScript_widgets#technique_1_roving_tabindex) to enable arrow-key navigation. Since we do not have such JS in the `menu.astro` page, we were using `tabindex=0` everywhere. You can either re-add `tabindex=0` to ensure that we can test visual focus states, or write some JS to implement roving tabindex.

Reply:
I have added `tabindex=0` back to the items.